content/projects/web-development-website/_index.md

```diff
@@ -0,0 +1,82 @@
+---
+content_type: project
+flavours:
+- none
+learning_outcomes:
+- web_dev_hosting
+- web_dev_responsive_page_design
+- web_dev_web_page_structure
+- web_dev_web_design_principals
+prerequisites:
+  hard:
+  - topics/free-code-camp/responsive-web-design/1-basic-html-and-html-5
+  - topics/free-code-camp/responsive-web-design/2-basic-css
+  - topics/what-to-put-in-your-cv
+  - topics/web-frontend/intro-to-web-design/
+  - projects/simple-website
+ready: true
+story_points: 3
+submission_type: link
+tags:
+- html
+- css
+- responsive web design
+title: Web Development - Build your first personal website
+---
+
+Now that you have finished creating your first simple personal portfolio, make it responsive and ensure that it passes through a linter.
+
+
+## Instructions
+
+The HTML and CSS should be developed in a logical way with good naming conventions that would allow another developer to easily understand and build upon your code. There is something called "Semantic Markup" - read all about it [here](https://seekbrevity.com/semantic-markup-important-web-design/).
+
+You can also make use of this cool [markup validation tool](https://validator.w3.org/) to validate the HTML code used in your website.
+
+Your web page files and folders should be organized and named in a logical manner.
+
+Once you have finished your design and are satisfied with the style and content, please submit a link of the website hosted on GitHub Pages. You must have hosted your simple portfolio to [Github Pages](https://guides.github.com/features/pages/) by now, please submit the same link of your website with the updated features. DO NOT submit a link to your github repo. We don't want to clone your website, we want to look at what you built.
+
+### Requirements:
+
+The requirements of the website remain the same:
+
+- Website can be a single page or have multiple pages (the structure is up to you), but it must contain the following information:
+  - About me
+  - Skills (Your tech stack: the languages and tools you use)
+  - Projects/Portfolio (for now this might be a bit empty, but if you've worked on anything cool then tell us about it)
+  - Contact (social media, email, phone, etc)
+- Host your site as a Github Page. In your README file include a link to your github page for easy navigation.
+
+## Some advice
+
+- Website must be responsive to different screen sizes - use a mobile-first approach!
+  - Use CSS Grid (see tutorials below) to design your site and make it responsive.
+- Website must be developed using HTML and CSS Grid with a logical structure and naming conventions in your code.
+- You can also add some Javascript to make things more interactive.
+- Feel free to use icons, colours, pictures or organograms if you need to.
+- Make sure that your html passes through a [linter](https://validator.w3.org/). It is important to reduce errors and improve the overall quality of your code.
```

Comment:
Is this advice or a requirement? Following advice is optional, meeting requirements is required.

Reply:
can you have a look at this - i think it got missed